HISTORY.rst

```diff
@@ -5,10 +5,21 @@ History
 0.30.0 (unreleased)
 -------------------
 
-Fixes
-~~~~~
+Breaking changes
+~~~~~~~~~~~~~~~~
+* All "Anuclim" indices and indicators have lost their ``src_timestep`` argument. Most of them were already not using it and now every function infers the frequency from the data directly. This may add stricter constraints on the time coordinate, the same as for :py:func:``xr.infer_freq``.
```

Comment:
```suggestion
* All "Anuclim" indices and indicators have lost their ``src_timestep`` argument. Most of them were not using it and now every function infers the frequency from the data directly. This may add stricter constraints on the time coordinate, the same as for :py:func:``xr.infer_freq``.
```